tests/h/traversal/roots_test.py

```diff
@@ -509,6 +511,14 @@ def test_it_fetches_the_requested_user(
 
         user_service.fetch.assert_called_once_with("acct:bob@example.com")
 
+    def test_it_fails_with_bad_request_with_invalid_ids(
```

Comment:
Maybe singular for `id`?

Reply:
Yep that makes more sense. 

It's also interesting because I think it reveals how I think about unit-tests. This test actually checks that 'it' does it once, but in my head it's making sure 'it' does it every time it's called in future (thus the plural).